### PR TITLE
Rename plainJava->javac

### PR DESCRIPTION
**Pull request by vparfonov**
**_1 Upvote**_ Signed-off-by: Vitaly Parfonov vparfonov@codenvy.com
@TylerJewell @slemeur @stour

### CODE REVIEW

**Comment by TylerJewell** [comment]
Ok

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/action/MarkDirectoryAsGroup.java [hunk 1]
```diff
@@ -42,6 +42,6 @@ public void update(ActionEvent e) {
         if (currentProject == null) {
             e.getPresentation().setVisible(false);
         }
-        e.getPresentation().setVisible("plainJava".equals(currentProject.getProjectConfig().getType()));
+        e.getPresentation().setVisible("javac".equals(currentProject.getProjectConfig().getType()));
```
Comment (benoitf) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/action/MarkDirectoryAsGroup.java]: why we're not using contants from PlainJavaProjectConstants ?

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/libraries/LibEntryPresenter.java [hunk 2]
```diff
@@ -83,7 +83,7 @@ public boolean isDirty() {
     @Override
     public void go(final AcceptsOneWidget container) {
         ProjectConfigDto projectConfig = appContext.getCurrentProject().getProjectConfig();
-        isPlainJava = "plainJava".equals(projectConfig.getType());
+        isPlainJava = "javac".equals(projectConfig.getType());
```
Comment (benoitf) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/libraries/LibEntryPresenter.java]: why we're not using contants from PlainJavaProjectConstants ?

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/sources/SourceEntryPresenter.java [hunk 3]
```diff
@@ -81,7 +81,7 @@ public boolean isDirty() {
     @Override
     public void go(AcceptsOneWidget container) {
         ProjectConfigDto projectConfig = appContext.getCurrentProject().getProjectConfig();
-        isPlainJava = "plainJava".equals(projectConfig.getType());
+        isPlainJava = "javac".equals(projectConfig.getType());
```
Comment (benoitf) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/sources/SourceEntryPresenter.java]: why we're not using contants from PlainJavaProjectConstants ?

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/sources/SourceEntryPresenterTest.java [hunk 4]
```diff
@@ -58,8 +58,8 @@
  */
 @RunWith(GwtMockitoTestRunner.class)
 public class SourceEntryPresenterTest {
-    final private static String PLAIN_TYPE = "plainJava";
-    final private static String SOURCE     = "lib";
+    final private static String JAVAC  = "javac";
```
Comment (benoitf) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/sources/SourceEntryPresenterTest.java]: why redefining constant defined in PlainJavaProjectConstants ?

**Comment by vparfonov** [comment]
@svor

**Comment by svor** [comment]
ok

File: plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/sources/SourceEntryPresenterTest.java (hunk 4 above)
Comment (vparfonov) [inline plugins/plugin-java/che-plugin-java-ext-lang-client/src/test/java/org/eclipse/che/ide/ext/java/client/project/classpath/valueproviders/pages/sources/SourceEntryPresenterTest.java]: it declared in other module, so for now o want to keep it as is after release we will fix this

**Comment by benoitf** [comment]
-1 constant needs to be used instead of duplicating values

**Comment by vparfonov** [comment]
@benoitf check now

**Comment by benoitf** [comment]
+1

**Pull request merged**